ui/src/i18n/en.json

```diff
@@ -92,6 +92,9 @@
         "newPassword": "New Password",
         "changePassword": "Change Password?"
       },
+      "updated": "User updated",
```

Comment:
Can these be under `resources.user.notification.*` ? It makes things more structured.